src/test/parse.ts

```diff
@@ -114,6 +118,25 @@ export function parseErrorsFromMarkup(text: string): LintError[] {
     return lintErrors;
 }
 
+function substituteMessage(substitutions: Map<string, string>, message: string): string {
```

Comment:
Re substitutions: It's not really a templateMap, it's a map of substitutions that may happen to contain a template

Re message: It's not necessarily a templateCall. It may be an identifier of a substitution, a "template call" or just plain text

I'd prefer leaving the names that way.